forecast.js

```diff
@@ -0,0 +1,41 @@
+var request = require('request');
+
+module.exports = function (location) {
+  return new Promise (function (resolve, reject) {
+    var apiKey = process.env.API_KEY;
+    console.log(location);
+    var url = `http://api.openweathermap.org/data/2.5/forecast?q=${encodeURIComponent(location)}&mode=json&units=imperial&APPID=${apiKey}`;
```

Comment:
removed the ',us' after the location to allow for searches from outside the United States